src/workers/parquet-exporter.ts

```diff
@@ -488,34 +494,45 @@ const exportToParquet = async ({
   tableName: string;
   startHeight: number;
   endHeight: number;
-  maxFileRows: number;
-}): Promise<void> => {
+  maxFileRows?: number;
+}): Promise<HeightRange[]> => {
   if (!existsSync(outputDir)) {
     mkdirSync(outputDir, { recursive: true });
   }
 
+  const heightRanges: HeightRange[] = [];
   let { minHeight, maxHeight } = await getHeightRange(db, tableName);
   minHeight = minHeight > BigInt(startHeight) ? minHeight : BigInt(startHeight);
   maxHeight = maxHeight < BigInt(endHeight) ? maxHeight : BigInt(endHeight);
   let rowCount = 0n;
+  let currentRangeStart = minHeight;
 
   for (let height = minHeight; height <= maxHeight; height++) {
     const heightRowCount = await getRowCountForHeight(db, tableName, height);
     rowCount += heightRowCount;
 
-    if (rowCount >= maxFileRows || height === maxHeight) {
-      const fileName = `${tableName}-minHeight:${minHeight}-maxHeight:${height}-rowCount:${rowCount}.parquet`;
+    if (
+      (maxFileRows !== undefined && rowCount >= maxFileRows) ||
+      height === maxHeight
+    ) {
+      heightRanges.push({
+        startHeight: currentRangeStart,
+        endHeight: height,
+        rowCount,
+      });
+
+      const fileName = `${tableName}-${currentRangeStart}-${height}-rowCount:${rowCount}.parquet`;
```

Comment:
We should keep the `minHeight:` and `maxHeight:` prefixes if possible. Did you run into an issue importing with those? I think it should work okay.

Reply:
It will works as `sed 's/blocks-//;s/-rowCount:[0-9]*\.parquet//'` from https://github.com/ar-io/ar-io-node/blob/b30e8d2ba9d7794c31a9cb30ef23dbe68620b3c6/scripts/clickhouse-import#L9 will get anything between `blocks-` and `-rowCount`, but I can add `minHeight:` and `maxHeight:` to keep it more readable

Reply:
Yeah, I'd like to keep them so we can more easily parse them out of the file name in the future.